src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -112,7 +112,7 @@ public bool MatchesPattern(string text, string pattern, CultureInfo culture)
             if (!culture.Equals(EnUSCultureInfo))
             {
                 patternMatcher = this.GetPatternMatcher(pattern, EnUSCultureInfo);
-                match = patternMatcher.GetFirstMatch(completionItemText);
+                match = patternMatcher.GetFirstMatch(completionItemText, includeMatchSpans);
```

Comment:
❗️ This change is a different bug and needs its own regression test. An acceptable alternative for this pull request would be to explicitly pass `false` (preserve original behavior) and create a separate bug to address the issue later.